.travis.yml

```diff
@@ -3,32 +3,26 @@ language: cpp
 jobs:
   include:
     - os: linux
-      env: OPENCV_VERSION="2.4.13.6" BITS_64=ON BITS_32=OFF CONFIGURATION_TYPE="Release" BUILD_NFIQ2_CLI=ON
+      env: BITS_64=ON BITS_32=OFF CONFIGURATION_TYPE="Release" BUILD_NFIQ2_CLI=ON
       compiler: gcc
     - os: linux
-      env: OPENCV_VERSION="3.4.8" BITS_64=ON BITS_32=OFF CONFIGURATION_TYPE="Release" BUILD_NFIQ2_CLI=ON
-      compiler: gcc
-    - os: linux
-      env: OPENCV_VERSION="4.4.0" BITS_64=ON BITS_32=OFF CONFIGURATION_TYPE="Release" BUILD_NFIQ2_CLI=ON
-      compiler: gcc
-    - os: linux
-      env: OPENCV_VERSION="4.4.0" BITS_64=ON BITS_32=OFF CONFIGURATION_TYPE="Release" BUILD_NFIQ2_CLI=ON
+      env: BITS_64=ON BITS_32=OFF CONFIGURATION_TYPE="Release" BUILD_NFIQ2_CLI=ON
       compiler: clang
     - os: osx
-      osx_image: xcode11.3
-      env: OPENCV_VERSION="2.4.13.6" BITS_64=ON BITS_32=OFF CONFIGURATION_TYPE="Release" BUILD_NFIQ2_CLI=ON
+      osx_image: xcode12.2
+      env: BITS_64=ON BITS_32=OFF CONFIGURATION_TYPE="Release" BUILD_NFIQ2_CLI=ON
       compiler: clang
     - os: osx
-      osx_image: xcode11.3
-      env: OPENCV_VERSION="3.4.8" BITS_64=ON BITS_32=OFF CONFIGURATION_TYPE="Release" BUILD_NFIQ2_CLI=ON
+      osx_image: xcode11.7
```

Comment:
xcode11.7 doesn't seem to exist in this list: https://docs.travis-ci.com/user/reference/osx/#using-macos